signup/signup.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>SIGN UP</title>
```

Comment:
Indentation should be of 2 space only. Please configure your editor with required specifications.